### DISCUSSION TITLE
Sorting criteria strings shouldnt use the web's translation

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#17995] [Q&A] [answered]
<img src="https://user-images.githubusercontent.com/86934170/165485767-69c5eee7-6d95-430a-b1ad-304aac5595e1.png" width="350" />
<img src="https://user-images.githubusercontent.com/86934170/165486201-8d2b216d-03dc-4885-a50c-3b441960b08a.png" width="350" />
<img src="https://user-images.githubusercontent.com/86934170/165486289-fca4b858-6e46-484c-a275-8fdc5be3381a.png" width="350" />
<img src="https://user-images.githubusercontent.com/86934170/165486349-1f3e354e-591a-4805-a763-c5f6a458f791.png" width="350" />
In Russian language, at least, "Sort by" thing has to be declensioned word after it, which means that a word changes its ending. 
For example: "Source" is "Источник", but with "Sort by" it should be "ИсточникУ" (added "У" letter) etc.
Without this change words wont be look gramatically correct.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/18014
